advanced_guides/asynchronous_updates.md

```diff
@@ -25,21 +25,21 @@ sequenceDiagram
   M->>-Q: dequeue update 2
 </mermaid>
 
-### Which actions are async?
+### Which updates are async?
```

Comment:
```suggestion
### Which actions are async?
```
or which `process, operation`.  `Which update` is not correct in context.

Reply:
I prefer `operation`.

Reply:
Would you prefer operation on the above change requests too ?

Reply:
Yes, it seems better.